src/components/Button/ButtonAddPostTopic.js

```diff
@@ -7,6 +7,7 @@ import {colors} from '../../utils/colors';
 import dimen from '../../utils/dimen';
 import {normalize} from '../../utils/fonts';
 import ShadowFloatingButtons from './ShadowFloatingButtons';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `COLORS` from the theme file is a good change as it replaces hardcoded color values with dynamic ones, improving modularity and consistency. 
```diff
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->